src/Shelly1/shelly_init.cpp

```diff
@@ -15,13 +15,20 @@
  * limitations under the License.
  */
 
+#include "mgos_hap.h"
+
 #include "shelly_hap_garage_door_opener.hpp"
+#include "shelly_hap_temperature_sensor.hpp"
 #include "shelly_input_pin.hpp"
 #include "shelly_main.hpp"
-#include "shelly_temp_sensor_ntc.hpp"
+#include "shelly_temp_sensor_ow.hpp"
+
+#define NUM_SENSORS_MAX 3
```

Comment:
this refers to the number of onewire sensors, so let's be specific: `MAX_NUM_OW_SENSORS`

Reply:
not necessarily: if implementing something like https://github.com/mongoose-os-apps/shelly-homekit/issues/384 which would need more or less than around 3 lines of code now, the limit of sensors would still hold because of the static config - how do you see this?

Reply:
made it clearer, to match the max sensors now

Reply:
Made it clearer to match the above explanation